<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>821</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1057</width>
    <height>821</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1057</width>
    <height>821</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 170, 206, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 15, 159);
font: 75 28pt &quot;Comic Sans MS&quot;;
</string>
    </property>
    <property name="text">
     <string>Búsqueda de empleos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(150, 129, 255);
font: 60 18pt &quot;Rockwell&quot;;
border-color: rgb(180, 126, 255);</string>
    </property>
    <property name="text">
     <string>Análisis 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(150, 129, 255);
font: 60 18pt &quot;Rockwell&quot;;
border-color: rgb(180, 126, 255);</string>
    </property>
    <property name="text">
     <string>Análisis 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>510</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(150, 129, 255);
font: 60 18pt &quot;Rockwell&quot;;
border-color: rgb(180, 126, 255);</string>
    </property>
    <property name="text">
     <string>Análisis 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>210</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(150, 129, 255);
font: 60 18pt &quot;Rockwell&quot;;
border-color: rgb(180, 126, 255);</string>
    </property>
    <property name="text">
     <string>Análisis 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(150, 129, 255);
font: 60 18pt &quot;Rockwell&quot;;
border-color: rgb(180, 126, 255);</string>
    </property>
    <property name="text">
     <string>Análisis 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(150, 129, 255);
font: 60 18pt &quot;Rockwell&quot;;
border-color: rgb(180, 126, 255);</string>
    </property>
    <property name="text">
     <string>Análisis 6</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1057</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
